config/locales/en.yml

```diff
@@ -8,3 +8,6 @@ en:
       success: "Wishlist was successfully deleted."
     update:
       success: "Wishlist was successfully updated."
+  items:
+    create:
+      success: "Item was successfully created!"
```

Comment:
Shame on You. :D Set Your editor properly to make an empty line in the end of file.